app/components/shared/vote_button_component.rb

```diff
@@ -35,14 +35,14 @@ def default_options
     end
 
     def vote
-      @vote ||= Vote.find_or_initialize_by(votable: votable, voter: current_user, vote_flag: parsed_value)
+      @vote ||= Vote.find_or_initialize_by(votable: votable, voter: current_user, vote_flag: parsed_value?)
     end
 
     def already_voted?
       vote.persisted?
     end
 
-    def parsed_value
+    def parsed_value?
```

Comment:
`parsed_value?` sounds like we're asking whether the value has been parsed :thinking:. What do you think about:

```suggestion
    def in_favor?
```